pkg/volume/azure_file/azure_provision.go

```diff
@@ -224,11 +240,11 @@ func (a *azureFileProvisioner) Provision(selectedNode *v1.Node, allowedTopologie
 }
 
 // Return cloud provider
-func getAzureCloudProvider(cloudProvider cloudprovider.Interface) (azureCloudProvider, error) {
+func getAzureCloudProvider(cloudProvider cloudprovider.Interface) (azureCloudProvider, string, error) {
```

Comment:
the default resource group is in `azureCloudProvider`, so it's not necessary to return `resourceGroup` here.

Reply:
Acctually, `azureCloudProvider` is an interface. We can not directly get the `resourceGroup`. Need to make a type assertion and judge the result firstly. Only in the function which has handled like this can we get `resourceGroup`. Otherwise, out of the function, need to assert type again. 

So in order to reduce the duplicated code, I put the `resourceGroup` into return values.